.travis.yml

```diff
@@ -12,11 +12,12 @@ gemfile:
   - gemfiles/rails_4.gemfile
   - gemfiles/rails_5.gemfile
 script:
-- bundle exec rspec
-- bundle exec codeclimate-test-reporter
+  - bundle exec rspec
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: 2.4.0
       gemfile: gemfiles/rails_3.gemfile
-
+addons:
+  code_climate:
+    repo_token: b65d6447fdd48601786f688ef382bd56fd597b906943ff5c27d3d0fbf1bfc966
```

Comment:
Looks like the missing commit status issues resolved themselves :)

The magic Bash script is explicitly [not required](https://github.com/codecov/example-ruby) for Travis.
The reason this was initially not working is that Codecov gets confused by public _org_ repos. They need to be given access manually in the org's integration settings for Codecov:

(at `https://github.com/organizations/deliveroo/settings/installations/7537`)

![](https://www.dropbox.com/s/9c87ijy5p0nbm84/Screen%20Shot%202017-03-06%20at%2009.29.34.png?raw=1)